templates/ldap.conf.j2

```diff
@@ -0,0 +1,60 @@
+<LDAP>
+	# LDAP server URL
+	URL		{{ ldap.url }}
+
+	{% if ldap.anonymous_bind == false %}
+	# Bind DN (If your LDAP server doesn't support anonymous binds)
+	BindDN		{{ ldap.bind_dn }}
+	# Bind Password
+	Password	{{ ldap.bind_password }}
+    {% endif %}
```

Comment:
It's weird. In my IDe I have no spaces at all. I don't know why the Github preview is showing one.
![image](https://cloud.githubusercontent.com/assets/2573405/21003537/30d44bb0-bd2c-11e6-8cf2-eba455e8fa71.png)